src/components/find-on-shelf/FindOnShelfModal.tsx

```diff
@@ -130,6 +158,16 @@ const FindOnShelfModal: FC<FindOnShelfModalProps> = ({
         <h2 className="text-header-h2 modal-find-on-shelf__headline">
           {`${title} / ${author}`}
         </h2>
+        {manifestations.some((manifestation) => {
+          return manifestation.materialTypes.some((materialType) => {
+            return materialType.specific.includes("periodikum");
+          });
+        }) && (
```

Comment:
Consider moving this logic out of the template.

See [this](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/172/files#diff-a77066a65eabdc34863f418f9a8581b8c603f5c709b44ac0253d38c4d78617b8R78) for inspiration.

Reply:
Great for improving readability of the HTML part of the component. Thanks!